components/light/index.rst

```diff
@@ -563,12 +563,15 @@ currently active light color.
           - addressable_scan:
               name: Scan Effect With Custom Values
               move_interval: 100ms
+              scan_length: 15
```

Comment:
```suggestion
              scan_length: 1
```

Use the default value here for consistency.

Reply:
Done.